src/applications/combined-debt-portal/debt-letters/components/HowDoIPay.jsx

```diff
@@ -1,114 +1,94 @@
-              {userData.deductionCode}
-            </li>
-          </ul>
-        ) : (
-          <ul>
-            <li>File Number</li>
-            <li>Payee Number</li>
-            <li>Person Entitled</li>
-            <li>Deduction Code</li>
-          </ul>
-        )}
-
-        <a className="vads-c-action-link--blue" href="https://www.pay.va.gov/">
-          Pay on pay.va.gov
-        </a>
-      </va-accordion-item>
-      <va-accordion-item header="Option 2: Pay by phone" id="second">
-        <p>
-          Call us at{' '}
-          <va-telephone
-            className="vads-u-margin-left--0p5"
-            contact="8008270648"
-          />
-          .
-        </p>
-        <p>
-          If calling internationally, use{' '}
-          <va-telephone contact="6127136415" international />.
-        </p>
-      </va-accordion-item>
-      {showDebtLetterDownload ? (
-        <va-accordion-item header="Option 3: Pay by mail" id="third">
-          <p className="vads-u-margin-y--0">
-            Find instructions on how to pay by mail in the demand letter sent to
-            your address or you can
-            <Link
-              className="vads-u-margin-left--0p5"
-              to="/debt-balances/letters"
-            >
-              download them online.
-            </Link>
-          </p>
-        </va-accordion-item>
+      <h3>Pay online</h3>
+      <p>
+        You can pay directly from your bank account or by debit or credit card
+        on the secure{' '}
+        <a
+          aria-label="Pay.gov - Opens in new window"
+          target="_blank"
+          href="https://www.pay.va.gov/"
+          rel="noreferrer"
+        >
+          pay.va.gov
+        </a>{' '}
+        website.
+      </p>
+      <p>
+        You will need to provide the following details to pay this debt online:
+      </p>
+      {userData ? (
+        <ul className="no-bullets">
+          <li>
+            <strong>File Number: </strong>
+            {userData.fileNumber}
+          </li>
+          <li>
+            <strong>Payee Number: </strong>
+            {userData.payeeNumber}
+          </li>
+          <li>
+            <strong>Person Entitled: </strong>
+            {userData.personEntitled}
+          </li>
+          <li>
+            <strong>Deduction Code: </strong>
+            {userData.deductionCode}
+          </li>
+        </ul>
       ) : (
-        <va-accordion-item header="Option 3: Pay by mail" id="third">
-          <p className="vads-u-margin-y--0">
-            Find instructions on how to pay by mail in the demand letter sent to
-            your address.
-          </p>
-        </va-accordion-item>
+        <ul>
+          <li>File Number</li>
+          <li>Payee Number</li>
+          <li>Person Entitled</li>
+          <li>Deduction Code</li>
+        </ul>
       )}
-    </va-accordion>
-  </article>
+
+      <a className="vads-c-action-link--blue" href="https://www.pay.va.gov/">
```

Comment:
Use the updated `<va-action-link>` component here: https://design.va.gov/storybook/?path=/docs/components-va-link-action--docs